.travis.yml

```diff
@@ -89,32 +92,33 @@ install:
   # I will remove the sphinx downgrade. Change setup.py requirements when it
   # be the case, Katta
   - echo "------- Install Dependencies -------"
-  - pip3 install -U pip setuptools
-  - pip3 install Jinja2 PyYAML pytest sympy --user
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-      pip3 install -U --user sphinx==2.4.4 nbsphinx>=0.3.2 m2r sphinx-rtd-theme jupyter;
-    fi
+  - pip install -U pip setuptools scikit-build Jinja2 PyYAML pytest "sympy<1.6"
 
 #=============================================================================
 # Build, test and install
 #=============================================================================
 script:
   - echo "------- Build, Test and Install -------"
-  - mkdir build && cd build
-  - cmake .. -DPYTHON_EXECUTABLE=$(which python3) -DCMAKE_INSTALL_PREFIX=$HOME/nmodl
-  - make -j 2
-  - make test
+  - mkdir build && pushd build
+  - PYTHONPATH=$(python -c "import sys; print(':'.join(sys.path))") cmake .. -DPYTHON_EXECUTABLE=$(which python) -DCMAKE_INSTALL_PREFIX=$HOME/nmodl
```

Comment:
I did not touch this. @Ohm?

Reply:
This is simply a portable way to get the python executable's absolute path. Otherwise you have to deal with osx and linux disagreeing on what way to best do it.